modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -868,6 +868,23 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
      */
     CV_EXPORTS_W Net readNetFromONNX(const String &onnxFile);
 
+    /** @brief Reads a network model <a href="https://onnx.ai/">ONNX</a> from
```

Comment:
There is a wrong order: "<a href="https://onnx.ai/">ONNX</a> from" but not "from <a href="https://onnx.ai/">ONNX</a>"

Reply:
@dkurt Good catch. Fixed.